routes.js

```diff
@@ -1,86 +1,86 @@
+    res.status(200).send(data.toString());
+  });
+});
 
 
 
-// createMyCustomApi.post('/addNewMember', (req, res) => {
-//   let data = fs.readFileSync("./members_Array.json"); //diavazoume to array pou exoume hdh
-//   let myObject = JSON.parse(data); // to knaoume js object me parsing
-//   let newData = req.body; // dilwnoume newData to reqbody apo postman
-//     myObject.push(newData); // pusharo sto yparoxn js object to body
-//       console.log(myObject);
-//     let newArray = JSON.stringify(myObject);
-//     fs.writeFile("./members_Array.json", newArray, err => {
-//       if(err) throw err;
-//       console.log(newArray)
-//     });
-//     res.sendStatus(200);
-// });
+createMyCustomApi.post('/addNewMember', (req, res) => {
+  let data = fs.readFileSync("./members_Array.json"); //diavazoume to array pou exoume hdh
+  let myObject = JSON.parse(data); // to knaoume js object me parsing
+  let newData = req.body; // dilwnoume newData to reqbody apo postman
+    myObject.push(newData); // pusharo sto yparoxn js object to body
+      console.log(myObject);
+    let newArray = JSON.stringify(myObject);
+    fs.writeFile("./members_Array.json", newArray, err => {
+      if(err) throw err;
+      console.log(newArray)
+    });
+    res.sendStatus(200);
+});
 
 
-// createMyCustomApi.put('/updateExistMember', (req, res) => {
-//   const {memberId, role , firstname, lastname} = req.body
-//   if(!memberId) {
-//     return res.status(400).send('Member ID is required');
-//   }
-//   const memberIndex = data.findIndex(member => member.memberId === memberId);
-//   if (memberId === -1) {
-//     return res.status(404).send("Member not found");
-//   }
-//   const updatedMember = {
-//     memberId,
-//     role : role,
-//     firstname : firstname ,
-//     lastname : lastname
-//   };
+createMyCustomApi.put('/updateExistMember', (req, res) => {
+  const {memberId, role , firstname, lastname} = req.body
+  if(!memberId) {
+    return res.status(400).send('Member ID is required');
+  }
+  const memberIndex = data.findIndex(member => member.memberId === memberId);
+  if (memberId === -1) {
+    return res.status(404).send("Member not found");
+  }
+  const updatedMember = {
+    memberId,
+    role : role,
+    firstname : firstname ,
+    lastname : lastname
+  };
 
-//   data[memberIndex] = updatedMember
+  data[memberIndex] = updatedMember
 
 
-//   saveData(data)
-//   res.status(201).send('Member updated successfully');
-// });
+  saveData(data)
+  res.status(201).send('Member updated successfully');
+});
 
 
 
-// let data = require('./members_Array.json'); // swsto datapath
+let data = require('./members_Array.json'); // swsto datapath
 
 
-// const saveData = (data) => {
-//   fs.writeFileSync('members_Array.json', JSON.stringify(data, null, 2)); // gia na grapsoume sto json arxeio
-// };
+const saveData = (data) => {
+  fs.writeFileSync('members_Array.json', JSON.stringify(data, null, 2)); // gia na grapsoume sto json arxeio
+};
```

Comment:
A bit surprising to see it in the middle of the code. Could you please move these lines to the top of the file?

Reply:
Done